README.md

```diff
@@ -1,215 +1,93 @@
 # FHIR Converter
 
-FHIR Converter is an open source project that enables conversion of health data from legacy formats to FHIR.
+FHIR Converter is an open source project that enables conversion of health data from legacy formats to and from FHIR.  The FHIR Converter uses the [Liquid template language](https://shopify.github.io/liquid/) and the .NET runtime.
```

Comment:
super nit: I believe Linishya wanted the 'c' to be lowercase.

Reply:
Okay, I can make that change.  It doesn't align with the repo name.

Reply:
Updated